frontend/src/app/components/modals/confirm-dialog/confirm-dialog.modal.ts

```diff
@@ -42,6 +42,8 @@ export interface ConfirmDialogOptions {
   closeByEscape?:boolean;
   showClose?:boolean;
   closeByDocument?:boolean;
+  passedData?:any;
+  classes?:{form?:string; button?:string};
 }
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/7cc3b1cd3e1f8ad3cf278c512d949802734aad14/frontend/src/app/components/modals/confirm-dialog/confirm-dialog.modal.ts#L48"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>